getting-started/ruby/main.rb

```diff
@@ -1,33 +1,139 @@
 # frozen_string_literal: true
 
-require "oso-cloud"
-require "sinatra"
-
-api_key = ENV["OSO_AUTH"]
-oso = OsoCloud::Oso.new(url: "https://cloud.osohq.com", api_key: api_key)
-
-helpers do
-  def protected!
-    @auth ||= Rack::Auth::Basic::Request.new(request.env)
-    if @auth.provided? and @auth.basic? and @auth.credentials
-      return @auth.credentials[0]
-    end
-    headers["WWW-Authenticate"] = 'Basic realm="restricted", charset="UTF-8"'
-    halt 401, "Not Authorized\n"
+require 'oso-cloud'
+require 'sinatra'
+
+# remember to update this before running the server!
+api_key = '<please provide your api key here>'
+oso = OsoCloud::Oso.new(url: 'https://cloud.osohq.com', api_key:)
```

Comment:
```suggestion
oso = OsoCloud::Oso.new(url: 'https://cloud.osohq.com', api_key: api_key)
```
(not sure if this some ruby shorthand i'm missing, ignore if so)

Reply:
Yes, this is ruby shorthand.